app/components/homeScreen/DailyReads.tsx

```diff
@@ -307,13 +314,20 @@ const DailyReads = (): any => {
       setActivityDataToShowInList(activityDataList)
 
     }
-  }, [activeChild?.uuid, activityTaxonomyId]);
+  }, [activeChild?.uuid, activityTaxonomyId,fetchAgain]);
   const keyExtractor = useCallback((item: any) => item?.id, []);
 
   const renderItem = useCallback((item: any, index: any, isAdvice: boolean) => {
     return <RenderDailyReadItem item={item} index={index} isAdvice={isAdvice} />;
   }, [favoriteAdvices, favoriteGames]);
 
+  const renderEmptyList = () => {
+      if(dataToShowInList?.length == 0 && !fetchAgain){

```

Comment:
Use strict equality to improve type safety: ===

Reply:
added strict equality